include/zephyr/usb/class/usb_hid.h

```diff
@@ -19,6 +19,9 @@
 extern "C" {
 #endif
 
+/** HID Specification release */
+#define USB_HID_VERSION		0x0111
```

Comment:
This header is API for the USB HID device implementation. Please move USB_HID_VERSION to `include/zephyr/usb/class/hid.h`.